src/backend/distributed/connection/connection_management.c

```diff
@@ -91,6 +92,26 @@ InitializeConnectionManagement(void)
 }
 
 
+/*
+ * InvalidateConnParamsHashEntries sets every hash entry's isValid flag to false.
+ */
+void
+InvalidateConnParamsHashEntries()
```

Comment:
I think some compilers complain when there is no `(void)` (though not sure which)

Reply:
Hm… I know that's true of declarations, but I thought it was OK to put in definitions. I'll fix it.